sdks/java/io/jms/build.gradle

```diff
@@ -30,14 +31,12 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.slf4j_api
   compile library.java.joda_time
-  compile "org.apache.geronimo.specs:geronimo-jms_1.1_spec:1.1.1"
```

Comment:
If nothing changed, please don't move lines around. It makes it harder to trace change history.